### ISSUE TITLE
Device completely resets after restart - every time!

### ISSUE THREAD

**Issue by powerslide24** [rg35xx-cfw/rg35xx-cfw.github.io#152]
I have a strange issue on my (otherwise flawless) RG35XX plus.

After the initial boot with a freshly flashed SD everything works fine and I am able to change/configure all settings (wifi, themes, sound, etc.).

After a restart, the system sets everything back to default - even the theme. Looks like a fresh system again - but Wifi not working anymore (at least not detecting networks by itself). If I change settings once again, everything starts from the beginning.

Tried it several times with different builds but it´s all the same again and again.

**Comment by waffledork** [comment]
Are you using the SD card it came with or a new one altogether? It sounds like a faulty SD card issue.

**Comment by AndreaCCIE** [comment]
I have this problem when there’s an sd in the slot 2. Once removed, it works

**Comment by powerslide24** [comment]
> Are you using the SD card it came with or a new one altogether? It sounds like a faulty SD card issue.

I never used the original one but tried it with two different branded cards (SanDisk and Samsung).

**Comment by powerslide24** [comment]
> I have this problem when there’s an sd in the slot 2. Once removed, it works

Yeah, it really looks like the 2nd card plays a certain role. I got it somehow running now - maybe after formatting the 2nd card directly with the handheld and adjusting the storage settings to „all external“. 

But before I had really strange symptoms with some themes kept and others not after a restart or these wifi issues. 

But thanks to everyone for your help 👍
